Dockerfile

```diff
@@ -27,13 +27,12 @@ WORKDIR $SRC
 
 RUN COMPONENT=${COMPONENT} make clean build
 
-FROM oraclelinux:7-slim
+FROM oraclelinux:8-slim
```

Comment:
To avoid any rate limiting when trying to access Docker Hub, I suggest replacing this with `FROM ghcr.io/oracle/oraclelinux:8-slim` instead. 

For context, the container images published on GitHub Container Registry are identical to those in the Docker Hub Official Library and can also be found on Oracle Container Registry. They're also multi-arch manifest lists, so they work on both `amd64` and `arm64v8`.

Apologies for not spotting this earlier.

Reply:
Updated